src/server/game/Entities/Object/Updates/UpdateFieldFlags.cpp

```diff
@@ -4987,7 +4976,6 @@ uint32 CorpseUpdateFieldFlags[CORPSE_END] =
     UF_FLAG_PUBLIC,                                         // OBJECT_FIELD_DATA+1
     UF_FLAG_PUBLIC,                                         // OBJECT_FIELD_DATA+2
     UF_FLAG_PUBLIC,                                         // OBJECT_FIELD_DATA+3
-    UF_FLAG_PUBLIC,                                         // OBJECT_FIELD_TYPE
```

Comment:
forgot to remove OBJECT_FIELD_DATA here

Reply:
this looks hand crafted to me, rejecting updatefield changes

Reply:
Yeah